Gemfile

```diff
@@ -24,6 +24,7 @@ gem "skylight" # automated performance testing https://www.skylight.io/
 gem "webpacker", "~> 5.4" # Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker
 gem "image_processing", "~> 1.12" # Set of higher-level helper methods for image processing.
 gem "lograge" # log less so heroku papertrail quits rate limiting our logs
+gem "filterrific", "~> 5.2.1" # filtering and sorting of models
```

Comment:
don't specifi gem version unless there is an error with a future version (and if so, add a comment about why it's pinned) because pinning prevents automatic upgrades

this gem was last updated in 2019 so I am not super enthusiastic about using it but it does seem stable

Reply:
Right, I'll remove the version specification.

I thought it was good practice to always pin the versions to make it more stable haha